README.md

```diff
@@ -1,2 +1,5 @@
 # TestCodeReview
 Test Code Review &amp; Pull Request With GitHub From VS
+
+#Kai
```

Comment:
Looks good but please also add one more line so that it has more content

Reply:
I will add and get back to you